contracts/src/v0.8/dev/CrossDomainDelegateForwarder.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./CrossDomainForwarder.sol";
+import "./interfaces/ForwarderInterface.sol";
+import "./interfaces/DelegateForwarderInterface.sol";
+
+/**
+ * @title CrossDomainDelegateForwarder - L1 xDomain account representation (with delegatecall support)
+ * @notice L2 Contract which receives messages from a specific L1 address and transparently forwards them to the destination.
+ * @dev Any other L2 contract which uses this contract's address as a privileged position,
+ *   can consider that position to be held by the `l1Owner`
+ */
+abstract contract CrossDomainDelegateForwarder is DelegateForwarderInterface, CrossDomainOwnable {
+
+}
```

Comment:
nit:

```suggestion
abstract contract CrossDomainDelegateForwarder is DelegateForwarderInterface, CrossDomainOwnable {}
```

Reply:
This is actually following the standard convention / generated output of `prettier-plugin-solidity`, which is applied consistently throughout the codebase.

Reply:
Ok, let's follow prettier here while it does seem inconsistent with empty constructors for example.

Reply:
For the sake of consistency, I'd like to keep to the same linting rules.